concierge/service/src/main/java/org/eclipse/ditto/concierge/service/starter/proxy/DefaultEnforcerActorFactory.java

```diff
@@ -79,6 +77,8 @@
     private static final String ENFORCER_CACHE_METRIC_NAME_PREFIX = "ditto_authorization_enforcer_cache_";
     private static final String ID_CACHE_METRIC_NAME_PREFIX = "ditto_authorization_id_cache_";
 
+    private static final Logger log = DittoLoggerFactory.getThreadSafeLogger(DefaultEnforcerActorFactory.class);
```

Comment:
I would suggest to remove this logger again as it is only used for 1 debug statement which does not really bring any benefit in a productive setup.